### ISSUE TITLE
Launcher keeps stopping

### ISSUE THREAD

**Issue by hakova** [GrapheneOS/os-issue-tracker#4121]
Hi,

This issue essentially made my phone unusable at the moment. The home screen keeps flashing due to constantly crashing launcher, and touchscreen does not respond to taps, etc. due to this. I can pull the notifications from the top of the screen that ends the flashing of the screen, but there is no way to be able to open an app since the launcher cannot stay open.  I am not sure if this issue is the same with #3159 or #3237 reported before, I suspect that it isn't. I cannot share the whole error message here because I couldn't figure out a way to get it from the phone to paste here due to unusability, but please see the snippet below that I hand-typed. Hopefully it gives some idea for troubleshooting:
`type: crash`
`osVersion: google/redfin/redfin:14/`
`UP1A.........`
`package: com.android.launcher3:34`
`process: com.android.launcher3`
`processUptime: 681 + 380 ms`
`java.time.DateTimeException: Invalid ID for offset-based ZoneIdL GMT+19:00 at java.time.ZoneId.ofWithPrefix(ZoneId.java:430) at java.time.ZoneId.of(ZoneId.java:400) at java.time.ZoneId.of(ZoneId.java:352) at java.time.ZoneId.of(ZoneId.java:308) at ......`

Also reported [here](https://discuss.grapheneos.org/d/16373-launcher-keeps-stopping)

**Comment by muhomorr** [comment]
Need more lines from the error report. Top lines that you've posted are not enough to find the place of the crash.

**Comment by hakova** [comment]
Thank you for the replies! This is how I solved the problem:
1. Cleared cache of the launcher, but it wasn't enough.
2. Cleared launcher storage, which reset the home screen(s) but it solved the issue.
3. Started adding apps, widgets to home screen one by one, and when I added the NOAA weather widget, the problem reappeared.
Therefore, I concluded that the widget mentioned above must have screwed up the time zone settings somehow. I am not using that widget anymore and things are back to normal. The app that the widget comes with (NOAA weather app) seems to behave well when opened.
